lib/Config.js

```diff
@@ -814,11 +835,12 @@ class Config extends EventEmitter {
                      * must become 'hosts'
                      */
                     host: process.env.S3KMIP_HOSTS,
-                    key: process.env.S3KMIP_KEY || undefined,
-                    cert: process.env.S3KMIP_CERT || undefined,
-                    ca: process.env.S3KMIP_CA
-                        ? process.env.S3KMIP_CA.split(',')
-                        : undefined,
+                    key: this._loadTlsFile(process.env.S3KMIP_KEY || undefined),
```

Comment:
should `_loadTlsFile` be called here?  this should be the filename or path?

Reply:
I don't see why it would throw an error, can you be more specific?

Reply:
does this comment apply to the other members of the tls struct? (e.g. ca and cert)

Reply:
you are correct. upon double checking, I have mixed up `this.kmip` vs `config.kmip`

Reply:
I did too, the reason for this PR...